run-tests.php

```diff
@@ -708,13 +708,13 @@ function main(): void
     if ($test_cnt) {
         putenv('NO_INTERACTION=1');
         usort($test_files, "test_sort");
-        $start_time = time();
+        $start_time = microtime(true);
```

Comment:
At this point might as well use ``hrtime()``

Reply:
Note that `hrtime` can be unavailable (returning false), but I don't know on what platforms.

Reply:
Huh, I thought it was always available

Reply:
```
#define ZEND_HRTIME_AVAILABLE (ZEND_HRTIME_PLATFORM_POSIX || ZEND_HRTIME_PLATFORM_WINDOWS || ZEND_HRTIME_PLATFORM_APPLE || ZEND_HRTIME_PLATFORM_HPUX || ZEND_HRTIME_PLATFORM_AIX)
```
I don't know which platforms are excluded from this :slightly_smiling_face: If they are supported by PHP then run-test.php should work on them. If there's no such platform the build system should error for missing hrtime instead.

Reply:
We have polyfill for the `hrtime()` function in this script so it's safe to use it: https://github.com/php/php-src/blob/9a34b57b289afe09036c860069c8d71adaab4940/run-tests.php#L806

Reply:
Ah indeed, I missed that.